src/components/ChecSlideoutPanel/Overlay.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
+    :class="`slideout-panel__overlay slideout-panel__overlay--${overlay}`"
+    @click="onOverlayDismiss"
+  />
+</template>
+
+<script>
+export default {
+  name: 'Overlay',
+  props: {
+    /**
+     * The overlay backdrop when the panel is opened. One of "light" and "dark".
+     */
+    overlay: {
+      type: String,
+      default: 'light',
+      validator(value) {
+        return ['light', 'dark'].includes(value);
+      },
+    },
+    /**
+     * Whether to close panel on overlay click. Default is true.
```

Comment:
```suggestion
     * Whether to close panel on overlay click. Default is false.
```

Or add `default: true` (the linter will complain if you do this though - maybe it needs to be `overlayClick` accepts strings e.g. `close` or `none`